test/utility/misc.c

```diff
@@ -139,6 +139,17 @@ testutil_cleanup(TEST_OPTS *opts)
 }
 
 /*
+ * testutil_check_long --
+ *	Check if WT_DISABLE_LONG_TESTS is set and exit if so
+ */
+void
+testutil_check_long(void)
+{
+	if (getenv("WT_DISABLE_LONG_TESTS") != NULL)
+		exit(0);
```

Comment:
I'm with @sulabhM I think a warning would be valuable here. If it isn't possible without causing problems for "make check" in this configuration, I'd be inclined to tie an output to the ``verbose`` option that is in the ``TEST_OPTS`` structure.

Having this function ``exit`` directly rather than return whether this is a long test, doesn't give the test case a chance to either cleanup after itself or alter it's behavior rather than being skipped altogether.